### ISSUE TITLE
Allow images in popups (sixel protocol?)

### ISSUE THREAD

**Issue by eugenesvk** [helix-editor/helix#6681]
I'd like to have a keybind that shows an image in a popup window, for example, a keymap cheat sheet on <kbd>F1</kbd>
As far as I'm aware, many terminals support a special standard imaging protocol https://www.arewesixelyet.com/ for that allows you to display images in the terminal

Would be great if I could control that behavior via a Helix command

**Comment by pascalkuthe** [comment]
I dont think we want to support feature like that from inside helix. There are multiple different nom-stanrd protocols. None of which are stadarized and usually only support by an emulator or two. Thisncoukd be done by plugins in the future but I dont see any reason why this needs to be integrated into core.

In the meantime.you can simply open images in a seperate window, with a terminal multiplexer or the terminal emulator (all of these usually support both overlaysnand splits)
